### DISCUSSION TITLE
How to use the system_open option to use other programs

### DISCUSSION THREAD

**Discussion by cirofabianbermudez** [nvim-tree/nvim-tree.lua#1928] [Q&A] [answered]
system_open = {
        cmd = "",
        args = {},
      },

**Comment by alex-courtis** [comment, marked as answer]
You can use any program you wish. The default on linux is `xdg-open`.

You could do something like this:
```lua
    system_open = {
      cmd = "vlc",
      args = { "--loop" },
    },
```

Updated help with defaults: https://github.com/nvim-tree/nvim-tree.lua/blob/13adc94e8ebde1d9ada2d131552cd4209c50ab8a/doc/nvim-tree-lua.txt#L540

**Reply by cirofabianbermudez** [reply]
It works perfectly thank you very much.
